pkg/reconciler/configuration/configuration_test.go

```diff
@@ -59,6 +61,16 @@ var revisionSpec = v1.RevisionSpec{
 func TestReconcile(t *testing.T) {
 	retryAttempted := false
 	now := time.Now()
+	clock := clock.NewFakeClock(now)
+
+	defaultDefaults, _ := cfgMap.NewDefaultsConfigFromMap(map[string]string{})
+	c := &cfgMap.Config{
+		Features: &cfgMap.Features{
+			ResponsiveRevisionGC: cfgMap.Enabled,
+		},
+		Defaults: defaultDefaults,
+	}
+	testCtx := cfgMap.ToContext(context.Background(), c)
 
 	table := TableTest{{
```

Comment:
I'm tempted to _not_ enable the flag here and have a separate test that has the feature enabled.

It'll make this test less noisy (I know it's too late).

Reply:
That's also the strategy I already took with v1/v2 gc_test, but here the change in functionality is relatively small. I could go either way with this one.